charts/timetagger/values.yaml

```diff
@@ -0,0 +1,60 @@
+
+## TimeTagger image configuration
+##
+image:
+  repository: ghcr.io/almarklein/timetagger
+  tag: v22.9.1
```

Comment:
Does this version number here mean that we need to update this file with each release? Can we avoid/automate that?